README.md

```diff
@@ -17,6 +17,37 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 2. Internal-only libraries SHOULD be prefixed with "graze-".
 3. It is RECOMMENDED to follow the style guide mantained in [agis-/git-style-guide](https://github.com/agis-/git-style-guide).
 
+## Logging
+
+1. Logs should be written to stderr and collected to a central store
+2. Application logs should be in [logfmt](https://brandur.org/logfmt) format in development and in logfmt or json for production `-- maybe always in logfmt?`
```

Comment:
Doesn't really look like sumologic has much knowledge on `logfmt` or they call it something else. So using json in aws would probably be better...